packages/spark-extras/components/dictionary/_dictionary.scss

```diff
@@ -0,0 +1,44 @@
+// ==================================================================
+// Dictionary
+// ==================================================================
+
+//
+// Available Variables
+///
+$dictionary-border: 1px solid $gray-medium;
+$dictionary-stripe-color: $gray-light;
+$dictionary-breakpoint: 38.4375em;
+
+.sprk-c-Dictionary--striped {
+  border: $dictionary-border;
+}
+
+.sprk-c-Dictionary__keyvaluepair {
+  width: 100%;
+  padding: $space-inset-m;
+
+  @media all and (min-width: $dictionary-breakpoint) {
+    display: table;
+    display: flex;
+    align-items: center;
+  }
+}
+
+// prettier-ignore
+.sprk-c-Dictionary--striped .sprk-c-Dictionary__keyvaluepair:nth-child(even) {
+  background: $dictionary-stripe-color;
```

Comment:
This comment might deserve the pedantic award haha so I wasn't going to even mention it but I thought maybe it would be a good discussion to bring up as we start to build stuff out. When we are declaring properties that have a shorthand option but we are only using one specific value - do we want to standardize on using the property that matches that value in that case? For this, we could use `background-color: ` instead of the shorthand `background`.  This is something that we could agree on as a team so that our code is consistent going forward. Again, this isn't the hugest deal and not something directly related to this specific PR. @yodasw16  - interested in your thoughts as well.

Reply:
I actually think this is a kind of big deal. I agree, we should be using background-color here. Shorthands like background can be problematic if someone writes css that comes later, it might not override it the way they expect.